docs/src/browsers.md

```diff
@@ -401,6 +401,23 @@ pytest test_login.py --browser-channel msedge
 dotnet test -- Playwright.BrowserName=chromium Playwright.LaunchOptions.Channel=msedge
 ```
 
+######
```

Comment:
Can there be a section with empty title? If so, we could use lang-specific paragraphs in more places.

Reply:
This seems common practise in our code-base.